benchexec/tools/btor2c.py

```diff
@@ -12,25 +12,28 @@
 
 class Tool(benchexec.tools.template.BaseTool2):
     """
-    Tool info for BTOR2C: A Converter from BTOR2 models to C programs
+    Tool info for Btor2C: A translator from Btor2 circuits to C programs
     """
 
     def executable(self, tool_locator):
-        return tool_locator.find_executable("btor2code", subdir="build")
+        return tool_locator.find_executable("btor2c", subdir="build")
```

Comment:
Rest of the changes look fine, but this is not backwards compatible. Was the old version only some preliminary beta, or was it publicly distributed and used? (Why not keep btor2c backwards compatible?)

Reply:
> Was the old version only some preliminary beta?

Yes, `btor2code` was just a preliminary version.
It was not distributed publicly.